src/autopas/containers/verletListsCellBased/verletListsCells/VerletListsCells.h

```diff
@@ -21,6 +21,7 @@
 #include "autopas/options/DataLayoutOption.h"
 #include "autopas/options/LoadEstimatorOption.h"
 #include "autopas/options/TraversalOption.h"
+#include "autopas/selectors/TraversalSelector.h"
```

Comment:
check if this does not produce a circular dependency

Reply:
Nope, we are fine there since `TraversalSelector` only includes traversals and no Containers. Also, the traversals do not include their containers.